README.md

```diff
@@ -12,4 +12,4 @@ and use it 3 times inside the `App` instead of static markup.
 ## Instructions
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_person/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://ColumbusDeveloper.github.io/react_person/) and add it to the PR description.
```

Comment:
Add the demo link to the PR description
<img width="687" alt="image" src="https://github.com/mate-academy/react_person/assets/64750958/c45298a4-a3d6-483d-8070-e9c0fe843587">